client/scripts/app.js

```diff
@@ -17,50 +50,46 @@ class ReviewSection extends React.Component {
 
           <div className='profileText'>
             <div className='profileInfo'>
-              <div className='name'>Zulema Z.</div>
+              <div className='name'>{this.state.name}</div>
               <br />
-              <div className='location'>Yoder, WY</div>
+              <div className='location'>{this.state.location}</div>
             </div>
 
             <div className='friends counterBox'>
               <img className='friendIcon icon' src='img/friendIcon.png' />
-              <div className='friendCount count'>123</div>
+              <div className='friendCount count'>{this.state.friendCount}</div>
               <div className='friendWord word'>friends</div>
             </div>
             <br />
 
             <div className='reviews counterBox'>
               <img className='reviewIcon icon' src='img/reviewIcon.png' />
-              <div className='reviewCount count'>456</div>
+              <div className='reviewCount count'>{this.state.reviewCount}</div>
               <div className='reviewWord word'>reviews</div>
             </div>
             <br />
 
             <div className='photos counterBox'>
               <img className='photoIcon icon' src='img/photoIcon.png' />
-              <div className='photoCount count'>789</div>
+              <div className='photoCount count'>{this.state.photoCount}</div>
               <div className='photoWord word'>photos</div>
             </div>
           </div>
         </div>
 
         <div className='review'>
           <div className='stars'>
-            <img className='star' src='img/starIcon.png' />
-            <img className='star' src='img/starIcon.png' />
-            <img className='star' src='img/starIcon.png' />
-            <img className='star' src='img/starIcon.png' />
+            <img className='star' src='img/nostarIcon.png' />
```

Comment:
Suggestion. Looks like individual star icons are being rendered. May be easier to save the images into S3 and call the url. Can render a star-review image based on what score the reviewer gave.